elevate-cpanel

```diff
@@ -909,9 +954,12 @@ sub run_stage_3 ($self) {
             # remove all ea- packages
             ssystem(q{/usr/bin/yum -y erase ea-*});
 
-            foreach my $rpm (qw/ yum-plugin-fastestmirror epel-release/) {
-                next unless Cpanel::Pkgr::is_installed($rpm);
-                ssystem( '/usr/bin/rpm', '-e', '--nodeps', $rpm );
+            # Leapp handles EPEL on CloudLinux
+            unless ( $self->upgrade_to_cloudlinux() ) {
```

Comment:
I see this pattern repeatedly. It seems like you have made some of these things auto-upgrade on cloudlinux. Do they auto-upgrade on almalinux and rocky? I'd be much more excited here if we'd fixed the overall problems of things like epel by making them recognized repositories. 

In all I feel like all of these conditionals are making the code a tangled unmaintainable mess.

Reply:
Things like EPEL do, in fact, upgrade on Alma/Rocky, they're in the recognized repository list now.
But these changes are only accessible from the testing channel at the moment - Leapp packages from the main channel don't have them in the list.

Reply:
When will they be in the production channel?